addons/mail/controllers/discuss/poll.py

```diff
@@ -0,0 +1,76 @@
+from markupsafe import Markup
+from werkzeug.exceptions import NotFound
+
+from odoo.addons.mail.tools.discuss import add_guest_to_context, Store
+from odoo.fields import Domain
+from odoo.http import Controller, route
+
+
+class PollController(Controller):
+    @route("/discuss/poll/create", type="jsonrpc", auth="user", methods=["POST"])
+    def poll_create(self, answers, channel_id, duration, question, allow_multiple_answers=False):
+        channel = self.env["discuss.channel"].search_fetch([("id", "=", channel_id)])
+        if not channel:
+            raise NotFound()
+        notification = Markup(
+            "<div class='o_mail_notification o-discuss-Poll-started'>%s</div>"
+        ) % self.env._("created a poll: %s.", Markup("<strong>%s</strong>") % question)
+        message = channel.message_post(
+            body=notification, message_type="notification", subtype_xmlid="mail.mt_comment"
+        )
```

Comment:
I don't think the notification is worth it: there's already an actual poll in message list and that's more than enough.

The notification for pinned message is useful because we could pin an old message and we must see in recent activities through the notification. This is not the case here, the poll is immediately recent activity by itself.